flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -161,69 +161,7 @@ run_test "Walkthrough Table Scala nightly end-to-end test" "$END_TO_END_DIR/test
 run_test "Walkthrough DataStream Java nightly end-to-end test" "$END_TO_END_DIR/test-scripts/test_datastream_walkthroughs.sh java"
 run_test "Walkthrough DataStream Scala nightly end-to-end test" "$END_TO_END_DIR/test-scripts/test_datastream_walkthroughs.sh scala"
 
-run_test "Test PubSub connector with Docker based Google PubSub Emulator" "$END_TO_END_DIR/test-scripts/test_streaming_gcp_pubsub.sh"
```

Comment:
I know some people use this file to run things locally; so this may break someone's workflow.

Reply:
I was considering setting up another file like this for the ci runs (but then it would be even harder to keep things in sync)

Given how aggressive the end to end test scripts modify things on the file system, I doubt that many people use these scripts.
That's why I decided for this approach.

How about keeping this file, and I comment out, instead of deleting?

Reply:
I now had 2 runs where all the end to end tests where passing.
If I have another successful run, I will include all end to end tests in this PR. Stay tuned :)